### ISSUE TITLE
Minor security vulnerability via babel-traverse 6.26.0

### ISSUE THREAD

**Issue by SiAdcock** [prebid/Prebid.js#12010]
## Type of issue
Security vulnerability

## Description
We maintain a [fork of Prebid.js](https://github.com/guardian/prebid.js) that is being flagged by Dependabot as containing a critical security vulnerability, introduced by `babel-traverse` (ID: CVE-2023-45133)

In Prebid.js, it looks to be coming from `babel-register` via:
```
babel-register@6.26.0 -> @babel/core@6.26.0 -> babel-traverse@6.26.0
```
[Relevant line in package.json](https://github.com/prebid/Prebid.js/blob/cda06f40bb1958f77888c0ae0bb486c5c55ac8d7/package.json#L59)

`babel-register` is used to [add Babel support](https://github.com/prebid/Prebid.js/blob/cda06f40bb1958f77888c0ae0bb486c5c55ac8d7/wdio.shared.conf.js#L19) to the [end-to-end testing task](https://github.com/prebid/Prebid.js/blob/cda06f40bb1958f77888c0ae0bb486c5c55ac8d7/gulpfile.js#L406).

It is reasonable to assume that the [end-to-end testing tasks](https://github.com/prebid/Prebid.js?tab=readme-ov-file#test-locally) (`gulp e2e-test`) are currently insecure. Do you agree, and if so, would it be possible to upgrade `babel-register@6.26.0` to `@babel/register@7.23.2` or higher? 

## Platform details

This affects at least versions v8.52.0 and v9 (latest) of Prebid.js

**Comment by patmmccann** [comment]
The alert says in bold: "Users that only compile trusted code are not impacted." 

You're welcome to PR the repo with the upgrade

**Comment by patmmccann** [comment]
Fixed in #12259
